pipeline/outputs/elasticsearch.md

```diff
@@ -8,54 +8,59 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
+| Logstash\_Prefix\_Key | When included: the value of the key in the record will be evaluated as key reference and overrides Logstash\_Prefix for index generation. If the key/value is not found in the record then the Logstash\_Prefix option will act as a fallback. The parameter is expected to be a [record accessor](../../administration/configuring-fluent-bit/classic-mode/record-accessor.md). |  | Yes |
+| Logstash\_Prefix\_Separator | Set a separator between logstash_prefix and date.| - | Yes |
+| Logstash\_DateFormat | Time format \(based on [strftime](http://man7.org/linux/man-pages/man3/strftime.3.html)\) to generate the second part of the Index name. | %Y.%m.%d | Yes |
+| Time\_Key | When Logstash\_Format is enabled, each record will get a new timestamp field. The Time\_Key property defines the name of that field. | @timestamp | Yes |
+| Time\_Key\_Format | When Logstash\_Format is enabled, this property defines the format of the timestamp. | %Y-%m-%dT%H:%M:%S | Yes |
+| Time\_Key\_Nanos | When Logstash\_Format is enabled, enabling this property sends nanosecond precision timestamps. | Off | Yes |
+| Include\_Tag\_Key | When enabled, it append the Tag name to the record. | Off | Yes |
+| Tag\_Key | When Include\_Tag\_Key is enabled, this property defines the key name for the tag. | \_flb-key | Yes |
+| Generate\_ID | When enabled, generate `_id` for outgoing records. This prevents duplicate records when retrying ES. | Off | Yes |
+| Id\_Key | If set, `_id` will be the value of the key from incoming record and `Generate_ID` option is ignored. |  | Yes |
+| Write\_Operation | The write\_operation can be any of: create (default), index, update, upsert. | create | Yes |
+| Replace\_Dots | When enabled, replace field name dots with underscore, required by Elasticsearch 2.0-2.3. | Off | Yes |
+| Trace\_Output | Print all elasticsearch API request payloads to stdout \(for diag only\) | Off | Yes |
+| Trace\_Error | If elasticsearch return an error, print the elasticsearch API request and response \(for diag only\) | Off | Yes |
+| Current\_Time\_Index | Use current time for index generation instead of message record | Off | Yes |
+| Suppress\_Type\_Name | When enabled, mapping types is removed and `Type` option is ignored. If using Elasticsearch 8.0.0 or higher - it [no longer supports mapping types](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html), so it shall be set to On. | Off | Yes |
+| Workers | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `2` | No |
+| Upstream | If plugin will connect to an _Upstream_ instead of a simple host, this property defines the absolute path for the Upstream configuration file, for more details about this refer to the [Upstream Servers](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md) documentation section. | | No |
```

Comment:
```suggestion
| Tag\_Key | When Include\_Tag\_Key is enabled, this property defines the key name for the tag. | `\_flb-key` | Yes |
| Generate\_ID | When enabled, generate `_id` for outgoing records. This prevents duplicate records when retrying ES. | `Off` | Yes |
| Id\_Key | If set, `_id` is the value of the key from incoming record, and `Generate_ID` option is ignored. | _none_ | Yes |
| Write\_Operation | The write\_operation can be any of: `create`, `index`, `update`, `upsert`. | `create` | Yes |
| Replace\_Dots | When enabled, replace field name dots with underscore, required by Elasticsearch 2.0-2.3. | `Off` | Yes |
| Trace\_Output | Print all elasticsearch API request payloads to stdout. For diagnostic use only. | `Off` | Yes |
| Trace\_Error | If elasticsearch return an error, print the elasticsearch API request and response. For diagnostic use only. | `Off` | Yes |
| Current\_Time\_Index | Use current time for index generation instead of message record. | `Off` | Yes |
| Suppress\_Type\_Name | When enabled, mapping types is removed and `Type` option is ignored. Elasticsearch 8.0.0 or later [doesn't support mapping types](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html), which requires this value to be `On`. | `Off` | Yes |
| Workers | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `2` | No |
| Upstream | If plugin will connect to an _Upstream_ instead of a simple host, this property defines the absolute path for the Upstream configuration file, for more details about this refer to the [Upstream Servers](../../administration/configuring-fluent-bit/classic-mode/upstream-servers.md) documentation section. | _none_ | No |
```

Reply:
Suggestion with small fix (there is no need to escape underscore with backslash if underscore is wrapped with backticks) and modification (`Off` and `On` can be considered special values and not literals, so there are not wrapped with backticks) was applied in c620db5a2ca5a0cd0f5a6474e0db8a0139bca3c3.